src/cli_plugin/install/downloaders/http.js

```diff
@@ -48,9 +47,20 @@ function downloadResponse({ resp, targetPath, progress }) {
 /*
 Responsible for managing http transfers
 */
-export default async function downloadUrl(logger, sourceUrl, targetPath, timeout) {
+export default async function downloadUrl(logger, sourceUrl, targetPath, timeout, proxy) {
+  let reqOptions = {
```

Comment:
I didn't use **const** because I add **proxy** key if proxy is defined (`reqOptions.proxy = proxy`). I'm not a js ninja, but is that ok with **const**?